src/views/Dogs/DogEdit.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import DogForm from '../../components/Dog/DogForm';
+
+test('should render an edit form', async () => {
+  const { container } = render(<DogForm />);
+
+  await screen.findByText('name:');
```

Comment:
Just a heads up, this is probably finding the label of the form, and not the input value. I believe findByDisplayValue would grab the form input value. 
https://testing-library.com/docs/queries/bydisplayvalue/